### PR TITLE
[FR] add misc new models/properties

### PR DESCRIPTION
**Pull request by kristapratico**

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,14 @@
...
   `list_document_classifiers`, `get_document_classifier`, and `delete_document_classifier`.
 - Added support for classifying documents on `DocumentAnalysisClient`: `begin_classify_document` and `begin_classify_document_from_url`.
+- Added model `QuotaDetails` and property `custom_neural_document_model_builds` on `ResourceDetails`.
+- Added kind `documentClassifierBuild` to `OperationSummary`.
+- Added property `expires_on` to `DocumentModelDetails` and `DocumentModelSummary`.
+- Added kind `formulaBlock` to `DocumentParagraph`.
+- Added property `common_name` to `DocumentKeyValuePair`.
+- Added property `code` to `CurrencyValue`.
+- Added properties `unit`, `city_district`, `state_district`, `suburb`, `house`, and `level` to `AddressValue`.
+- Added "boolean" `value_type` and `bool` `value` to `DocumentField`.
```
Comment (samvaity) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:20]: do you know if DocumentLanguage was added or separate PR?

**Review by samvaity — APPROVED** [review]
Changes lgtm.
Do we want to create a follow-up issue for current testing gaps?

File: sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py [hunk 2]
```diff
@@ -1999,6 +2006,19 @@ class AddressValue:
...
+    """Apartment or office number"""
+    city_district: Optional[str]
+    """Districts or boroughs within a city, such as Brooklyn in New York City or City
+    of Westminster in London."""
+    state_district: Optional[str]
+    """Second-level administrative division used in certain locales."""
+    suburb: Optional[str]
+    """Unofficial neighborhood name, like Chinatown."""
+    house: Optional[str]
+    """Build name, such as World Trade Center."""
```
Comment (witemple-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py:2019]:
```suggestion
    """Building name, such as World Trade Center."""
```

File: sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py [hunk 3]
```diff
@@ -1999,6 +2006,19 @@ class AddressValue:
     street_address: Optional[str]
     """Street-level address, excluding city, state, countryRegion, and
      postalCode."""
+    unit: Optional[str]
+    """Apartment or office number"""
```
Comment (witemple-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py:2010]:
```suggestion
    """Apartment or office number."""
```

**Review by witemple-msft — APPROVED** [review]
LGTM. Couple of typos.

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 1 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:20]: It looks like `DocumentLanguage` was added in the GA release. Did you mean a different type?

**Comment by kristapratico** [comment]
> Changes lgtm.
Do we want to create a follow-up issue for current testing gaps?

yes keeping note of our testing gap. Hoping to do better before Friday, we'll see 😟

**Pull request merged**